### ISSUE TITLE
Invalid command

### ISSUE THREAD

**Issue by json-m** [m13253/VxWireguard-Generator#1]
seems everything i try is an invalid command, has syntax changed vs the documented way?

```
$ vwgen add
vwgen: Invalid command 'add'

$ vwgen add wgtest
vwgen: Invalid command 'add'

$ vwgen add --help
vwgen: Invalid command 'add'
```

**Comment by m13253** [comment]
It is a bug.

Could you please provide your Python version and your OS version?

**Comment by json-m** [comment]
i tried using python3.6 on both alpine linux 3.7 and centos7

```
jason at a in ~
$ vwgen add
vwgen: Invalid command 'add'

jason at a in ~
$ cat /etc/*release
3.7.0
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.7.0
PRETTY_NAME="Alpine Linux v3.7"
HOME_URL="http://alpinelinux.org"
BUG_REPORT_URL="http://bugs.alpinelinux.org"
```

```
[root@23e81f24ce4d /]# python36 /usr/local/bin/vwgen add
vwgen: Invalid command 'add'
[root@23e81f24ce4d /]# cat /etc/*release
CentOS Linux release 7.5.1804 (Core)
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

CentOS Linux release 7.5.1804 (Core)
CentOS Linux release 7.5.1804 (Core)
```

**Comment by m13253** [comment]
Fixed in commit 01e497578744e0380c456c631be7ed24188fcea0.
Also fixed Python 3.6 incompatibility in commit 77ed7281d4a60123c61e3861c4b0a96cb765d812.

I have tested the code with both Python 3.6 and 3.7.
Please rebuild it with:
```
git pull
python3 setup.py build
sudo python3 setup.py install --force
```

Please let me know whether the problems are resolved.
And, please make sure you keep backups of the database before you test unstable code.

**Comment by json-m** [comment]
does appear to be resolved, thanks! 

closing issue.
